packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -42,3 +42,21 @@ class Checksum {
   @override
   int get hashCode => _checksums.hashCode;
 }
+
+/// Parses a VM snapshot dependency file.
+///
+/// Snapshot dependency files are a single line mapping the output snapshot to a
+/// space-separated list of input files used to generate that output. e.g,
+///
+/// outfile : file1.dart file2.dart file3.dart
+Set<String> readDepfile(String depfilePath) {
+  // Depfile format:
+  // outfile1 outfile2 : file1.dart file2.dart file3.dart
+  final String contents = fs.file(depfilePath).readAsStringSync();
```

Comment:
It used to be async at the original location though? https://github.com/flutter/flutter/pull/11084/files#diff-4692fef441cd0d0ee38dac80604a90b2L108